grader/src/main/java/edu/pdx/cs410J/grader/scoring/InvalidFileContentException.java

```diff
@@ -0,0 +1,10 @@
+package edu.pdx.cs410J.grader.scoring;
+
+/**
+ * Created by sushamkumar on 9/1/17.
+ */
+public class InvalidFileContentException extends Exception {
+    public InvalidFileContentException() { super("Out file is corrupted"); }
```

Comment:
Is this constructor ever called?  If not, let's delete it.

Reply:
No, this constructor is never called for now. But, I have kept this so that If there isn't any message to provide then we could use this constructor. I will remove this, so that custom message makes more sense.